src/t_zset.c

```diff
@@ -3432,6 +3432,12 @@ void genericZrangebylexCommand(zrange_result_handler *handler,
 
     handler->beginResultEmission(handler);
 
+    /* For invalid offset, return directly. */
+    if (offset >= (long)zsetLength(zobj)) {
```

Comment:
Is this just an optimization for early exit?

Reply:
Yes. same as above